### ISSUE TITLE
test problem

### ISSUE THREAD

**Issue by hagemon** [HYPJUDY/caffe-dnnh#4]
Hello
Thanks for your hard work.
After downloading your source code and datasets and deployed it in runtime/cifar_hash_dataset, I ran`./run_train.sh` in runtime/12bit and after serveral hours I got

> .......
> I0913 18:39:05.471935 10028 solver.cpp:311] Iteration 100000, loss = 0.000143271
> I0913 18:39:05.471941 10028 solver.cpp:316] Optimization Done.

But when I ran `./run_test.sh` after training, I got

> ......
> I0913 19:03:20.272930 12613 caffe.cpp:319] Running for iteration#589...
> I0913 19:03:20.296061 12613 caffe.cpp:354] Loss: 0
> Reading images's hashcode from files...
> Starting querying images in 58999 images' pool...
> Finish querying 1 images in 58999 pool images set within the top 10000 returned neighbors!
> Mean Average Precision(MAP): 0
> MAP of label#1: 0
> MAP of label#2: 0
> MAP of label#3: 0
> MAP of label#4: 0
> MAP of label#5: 0
> MAP of label#6: 0
> MAP of label#7: 0
> MAP of label#8: 0
> MAP of label#9: 0
> MAP of label#10: 0

I have not modified any code and don't know what I did wrong. Could you please give me some advices?
Thank you.
